configs/match-base.cfg

```diff
@@ -1,38 +0,0 @@
-# NOT MEANT TO BE USED DIRECTLY - include a compute/*.cfg file in addition to this one
```

Comment:
need to make `kubernetes/match.sh` not reference `match-base.cfg` since it's deleted

Reply:
done